src/routes/auth.js

```diff
@@ -33,6 +34,27 @@ router.post(
 	})
 )
 
+router.post(
+	'/invite-user',
```

Comment:
final thing for now @itzgeebee , let's validate the request like we do for the rest, to ensure we have the right data

Reply:
Okay sir